app/renderer/components/common/switchControl.js

```diff
@@ -36,29 +38,49 @@ class SwitchControl extends ImmutableComponent {
     >
       {
         this.props.leftl10nId && this.props.topl10nId
-        ? <div className='switchControlText'><div className='switchControlLeftText'><div className='switchSpacer'>&nbsp;</div><label className='switchControlLeftText' onClick={this.onClick} data-l10n-id={this.props.leftl10nId} /></div></div>
+        ? <div className='switchControlText'>
+          <div className='switchControlLeftText'>
+            <div className='switchSpacer'>&nbsp;</div>
+            <label className={cx({
+              [css(styles.switchControlText_left__label, this.props.small && styles.switchControlText__label_small)]: true,
+              [this.props.customLeftTextClassName]: !!this.props.customLeftTextClassName
+            })}
+              onClick={this.onClick} data-l10n-id={this.props.leftl10nId}
+            />
+          </div>
+        </div>
         : (this.props.leftl10nId
-          ? <label className='switchControlLeftText' onClick={this.onClick} data-l10n-id={this.props.leftl10nId} />
+          ? <label className={cx({
```

Comment:
Before rebasing this line had been wrapped with `div`. It was a regression, so I removed that.